content/core-features/presence.textile

```diff
@@ -8,3 +8,7 @@ index: 27
   <img src="/images/diagrams/Channels-Presence.gif" style="width: 100%" alt="Presence representation">
 </a>
  Furthermore, if persistence is enabled on the presence channel, you can also retrieve "presence history":/rest/history#presence-history for the channel, i.e, static data about historical presence states of your clients/devices. This operation also can be done using both Ably's "Realtime":/realtime and "REST":/rest libraries.
+
+h2(#occupancy-presence). Occupancy and presence
+
+"Occupancy":/core-features/occupancy differs from presence in that presence enables a client to identify themselves to other clients subscribed to presence events on a channel, as well as to update their member data for subscribers to view. Presence events are emitted when a client enters or leaves a channel or when they update their member data. Occupancy provides a count of the clients connected to a channel, broken down into specific metric categories. Occupancy data can be viewed at a specific point in time when using channel enumeration, or emitted as events when metrics change using an integration rule source or inband channel occupancy.
```

Comment:
"when metrics change using an integration rule source" - would it be better to refer explicitly to the `channel.occupancy` source?

Reply:
Yes, this makes it much more explicit thanks. Updated in https://github.com/ably/docs/pull/1232